internal/sbi/router.go

```diff
@@ -45,6 +45,10 @@ func newRouter(s *Server) *gin.Engine {
 	spyFamilyGroup := router.Group("/spyfamily")
 	applyRoutes(spyFamilyGroup, s.getSpyFamilyRoute())
 
+	kuroumiGroup := router.Group("/kuromi")
+	applyRoutes(kuroumiGroup, s.getKuromiRoute())
+	applyRoutes(kuroumiGroup, s.postKuromiRoute())
```

Comment:
combin to `getKuromiRoute()`.
The get here is just get, not "GET" in http method. 
You can refer to each NF